assets/js/amber.js

```diff
@@ -242,6 +242,7 @@ document.addEventListener("DOMContentLoaded", function () {
                 form.setAttribute("action", element.getAttribute("href"));
                 form.setAttribute("method", "POST");
                 input.setAttribute("type", "hidden");
+                input.setAttribute("name", "_method");
```

Comment:
Oh man. You changed the code from jquery to this so fast last night I just assumed it had everything and didn't look at the details. Sorry.

Reply:
Yeah, my bad, I missed this line, I guess I was tired too :sweat: